techniques/default-directive-names.conf

```diff
@@ -0,0 +1,22 @@
+
+#
+# This file contains the default name that a directive gets in Rudder UI creation pop-up.
+#
+# The file format is a simple "key:value" file, with key being the techniqueId
+# or techniqueId/version and the value being the name to use.
+#
+# An empty value will lead to an empty default name.
+# For a new Directive, we will try to lookup "TechniqueName/version" and if not
+# available "TechniqueName" from this file. If neither key is available, the
+# pop-up will use the actual Technique name as default.
+# 
+# Don't forget to commit the file in git (git add default-directive-names.conf; 
+# git commit -m "change new directive name) and reload technique library
+# (rudder server reload-techniques) to have modifications seen by Rudder. 
+#
+
+# Default pattern for new directive from "userManagement" technique:
+userManagement=User: Role <name>: <login>
```

Comment:
I think we should stay consistent, at least over core techniques, and provide better names.